setup-runner-linux.sh

```diff
@@ -47,12 +47,8 @@ if [[ -z "$username" ]]; then
   exit 1
 fi
 
-if [[ -z "$noreboot" ]]; then
-  noreboot=false
-fi
-
 if [[ -z "$runnerlabels" ]]; then
-  runnerlabels="experimental-ubuntu"
+  runnerlabels="azure-ex"
```

Comment:
Make this an input argument

Reply:
It is an input argument. This checks if the user has specified anything. If not, assign it a default value of "azure-ex."

Reply:
Oh, I see. Ignore me then. 😄